playbooks/roles/dev-patcher/defaults/main.yml

```diff
@@ -20,3 +20,5 @@ dev_patcher_patches:
   - 639861
   # osh-infra: Add support for mulitple VIPS
   - 640115
+  # osh: Usage of rally.conf for services with rally test for debugging
```

Comment:
related to this PR?

Reply:
will remove it as its not needed for this change.